### ISSUE TITLE
H (--header) custom header

### ISSUE THREAD

**Issue by ghost** [JohnWoodman/FES#4]
meg (tomnomnom) have -H (--header) custom header inside all http requests.
It's possible to add at fas ?

Another thing is, some books to learn rust to contribute ?

Cheers

**Comment by JohnWoodman** [comment]
Yeah it's definitely possible! I'm a little busy at the moment, so not sure when I'll have time to add it, but if you want to make a pull request, that would be great. Here's a great site that I learned more about Rust: https://www.rust-lang.org/learn
